patroni/dcs/etcd.py

```diff
@@ -596,3 +598,6 @@ def watch(self, leader_index, timeout):
             return super(Etcd, self).watch(None, timeout)
         finally:
             self.event.clear()
+
+
+etcd.EtcdError.error_exceptions[300] = EtcdRaftInternal
```

Comment:
What does this line achieve ?

Reply:
It adds it [here](https://github.com/jplana/python-etcd/blob/master/src/etcd/__init__.py#L279)